packages/es-components/config/import-matchers.js

```diff
@@ -0,0 +1 @@
+import 'jest-dom/extend-expect';
```

Comment:
I don't particularly like running them `afterEach` test. Running them in `beforeEach` makes more sense to me, and I only added it where I know it was a problem.

Reply:
How easy is it to identify that we need to call `cleanup()`? I haven't worked with this library yet so that's just something I don't fully understand. Maybe going forward providing a `beforeEach/afterEach` hook of some kind will simplify the development process by ensuring that the "state" is clean at the start of each test. If it's simple enough to tell then maybe the hook isn't necessary.

Reply:
You'll know pretty easily. If you run all the tests in a suite and they fail but they pass when run individually, it is most likely a race condition and the DOM is not in a clean state. I would be fine adding a hook if this becomes a problem for people. But I would do it `beforeEach` and not `afterEach`.

Reply:
Making a quick note here in case we do come back to this

```js
import { cleanup } from 'react-testing-library';
beforeEach(cleanup);
```